src/map/pc.c

```diff
@@ -5070,6 +5065,11 @@ int pc_useitem(struct map_session_data *sd,int n) {
 		return 0;
 	}
 
+	if (battle_config.storage_use_item == true && sd->state.storage_flag != STORAGE_FLAG_CLOSED) {
```

Comment:
Since `battle_config.storage_use_item` is an `int` and not a `bool`, can you change this check? (either `== 1` or `!= 0`, up to you)